### PR TITLE
Pair of graphics bugs

### PR DESCRIPTION
**Pull request by ivanpopelyshev**
One critical bug was introduced in #5964 
You can see it at https://pixijs.io/examples/#/graphics/advanced.js - lineTextureStyle doesn't work at all.
Color should be 0xffffff by default.

As for another minor performance bug, it survived several graphics PR's, its missing `!!style.native` for case when `native` is undefined (fillStyle). 

Right now it was affected if we had several fillStyles of the same color, or lineStyle/fillStyle of the same color - it added more batches in GraphicsGeometry.

I've added one test for it.

### CODE REVIEW

File: packages/graphics/src/Graphics.js [hunk 1]
```diff
@@ -322,7 +322,7 @@ export class Graphics extends Container
         options = Object.assign({
             width: 0,
             texture: Texture.WHITE,
-            color: 0x0,
+            color: 0xFFFFFF,
```
Comment (bigtimebuddy) [inline packages/graphics/src/Graphics.js]: What is this white? Black was intentional as a default.

Comment (eXponenta) [reply packages/graphics/src/Graphics.js]: Because it is tint for texture, and always was white. 
Black tint => black filled result, texture not showing.

Comment (bigtimebuddy) [reply packages/graphics/src/Graphics.js]: Ohhh, I didn't realized that `color` was a tint. The documentation needs to be updated.

Comment (ivanpopelyshev) [reply packages/graphics/src/Graphics.js]: oh, but in lineStyle default was black, right? eh, we might have a problem, need one more test

**Comment by ivanpopelyshev** [comment]
Done.

**Comment by bigtimebuddy** [comment]
Thanks @ivanpopelyshev

**Pull request merged**